src/sass/layouts/main/_hero.scss

```diff
@@ -1,14 +1,7 @@
 // Володимир
-.hero {
-    @media screen and (min-width:$desktop) {
-
-        padding-left: 110px;
-        height: 302px;
 
-    }
-}
 
-.hero-flex {
+.hero__flex {
     @media screen and (min-width:$tablet) {
         display: flex;
     }
```

Comment:
@media screen and (min-width:$desktop) {
        padding-top: 15px;
    }  

delete

Reply:
+